storybook/stubs/shared/stores/send/TransactionStore.qml

```diff
@@ -1,3 +1,270 @@
+        return ""
+    }
+
+    function getShortChainIds(chainIds) {
+        let listOfChains = chainIds.split(":")
+        let listOfChainIds = []
+        for (let k =0;k<listOfChains.length;k++) {
+            for(let i=0;i<NetworksModel.allNetworks.count;i++) {
+                if(NetworksModel.allNetworks.get(i).shortName === listOfChains[k]) {
+                    listOfChainIds.push(NetworksModel.allNetworks.get(i).chainId)
+                }
+            }
+        }
+        return listOfChainIds
+    }
+
+    function setSendType(sendType) {
+        root.sendType = sendType
+    }
+
+    function setSelectedRecipient(recipientAddress) {
+        root.selectedRecipient = recipientAddress
+    }
+
+    function setSelectedAssetSymbol(symbol) {
+       root.selectedAssetSymbol = symbol
+    }
+
+    function getWei2Eth(wei, decimals) {
+        return wei/(10**decimals)
+    }
+
+    function updateRoutePreferredChains(chainIds) {
+        root.toNetworksModel.updateRoutePreferredChains(chainIds)
+    }
+
+    function toggleShowUnPreferredChains() {
+        root.showUnPreferredChains = !root.showUnPreferredChains
+    }
+
+    property string amountToSend
+    property bool suggestedRoutesCalled: false
+    function suggestedRoutes(amount) {
+        root.amountToSend = amount
+        root.suggestedRoutesCalled = true
+    }
+
+    enum EstimatedTime {
+        Unknown = 0,
+        LessThanOneMin,
+        LessThanThreeMins,
+        LessThanFiveMins,
+        MoreThanFiveMins
+    }
+
+    function getLabelForEstimatedTxTime(estimatedFlag) {
+        switch(estimatedFlag) {
+        case TransactionStore.EstimatedTime.Unknown:
+            return qsTr("~ Unknown")
+        case TransactionStore.EstimatedTime.LessThanOneMin :
+            return qsTr("< 1 minute")
+        case TransactionStore.EstimatedTime.LessThanThreeMins :
+            return qsTr("< 3 minutes")
+        case TransactionStore.EstimatedTime.LessThanFiveMins:
+            return qsTr("< 5 minutes")
+        default:
+            return qsTr("> 5 minutes")
+        }
+    }
+
+    function getNetworkIcon(chainId) {
+        for(let i=0;i<NetworksModel.allNetworks.count;i++) {
+            if(NetworksModel.allNetworks.get(i).chainId.toString() === chainId.toString()) {
+               return NetworksModel.allNetworks.get(i).iconUrl
+            }
+        }
+    }
+
+    function resetStoredProperties() {
+        root.amountToSend = ""
+        root.sendType = Constants.SendType.Transfer
+        root.selectedRecipient = ""
+        root.selectedAssetSymbol = ""
+        root.showUnPreferredChains = false
+        root.fromNetworksModel.reset()
+        root.toNetworksModel.reset()
+    }
+
+
+    function getNetworkName(chainId) {
+        for(let i=0;i<NetworksModel.allNetworks.count;i++) {
+            if(NetworksModel.allNetworks.get(i).chainId.toString() === chainId.toString()) {
+               return NetworksModel.allNetworks.get(i).chainName
```

Comment:
Check ModelUtils, we have functions to do this kind of operations there :) It would greatly simplify this whole file